app/src/shared/stores/streamAbilities.ts

```diff
@@ -175,12 +183,14 @@ export function useCurrentStreamAbility(permission: StreamPermission) {
 
     const { address } = useAuthController().currentAuthSession
 
-    return (
-        useStreamAbility(streamId, address, permission) ||
-        (!streamId &&
-            (permission === StreamPermission.EDIT ||
-                permission === StreamPermission.GRANT))
-    )
+    const hasPermission = useStreamAbility(streamId, address == null || address.length === 0 ? undefined : address, permission)
+
+    if (!streamId) {
+        return (permission === StreamPermission.EDIT ||
+            permission === StreamPermission.GRANT)
+    }
+
+    return hasPermission
```

Comment:
Changed this so that it correctly passes through `undefined` permissions which are used for showing loading state.